media/CircleEditor/view-sidebar.js

```diff
@@ -1456,30 +1438,297 @@ sidebar.ArgumentView = class {
         return true;
     }
 
-    /**
-     * TODO Implement compareData()
-     * Compare and change transformed data against corrected data
-     */
+    compareData(bufferArr, originalArr, original, type) {
```

Comment:
`original` and `type` are too broad to think about the meaning.
Could you make this little bit more specific?
For example, if correct, like `originalBuf` or `arrType`